governance/xc_admin/packages/crank_executor/src/index.ts

```
@@ -213,5 +213,9 @@ async function run() {
 }
 
 (async () => {
-  await run();
+  try {
+    await run();
+  } catch (err) {
+    console.error(err);
+  }
```

this no longer throws on errors -- is that intentional?

---

i mean it's not just improving logging, it's also suppressing errors

---

I'll make it throw...

---

Without this trick the error wasn't getting printed